tests/functional/nix-profile.sh

```diff
@@ -191,7 +203,8 @@ nix profile install $flake2Dir --priority 0
 # flake references.
 # Regression test for https://github.com/NixOS/nix/issues/8284
 clearProfiles
-nix profile install $(nix build $flake1Dir --no-link --print-out-paths)
+nix build $flake1Dir --out-link result
+nix profile install $(readlink result)
```

Comment:
This was a bit of a nasty one. `nix build $flake1Dir --no-link --print-out-paths` has _2_ outputs: `flake1-man` and `flake1`. To make sure we only refer to a single output (`.out`) I've opted to output links and read the correct link.

Reply:
You should be able to use `nix build $flake1Dir^out` for that

Reply:
Aah, right, forgot about the syntax. Thanks!